packages/fx-core/resource/package.nls.json

```diff
@@ -321,7 +321,7 @@
   "core.createProjectQuestion.projectType.customCopilot.title": "App Features Using Teams AI Library",
   "core.createProjectQuestion.projectType.customCopilot.placeholder": "Select an option",
   "core.createProjectQuestion.projectType.copilotHelp.label": "Don't know how to start? Use GitHub Copilot Chat",
-  "core.createProjectQuestion.projectType.copilotHelp.detail": "Chat with GitHub Copilot and get step-by-step instructions to develop your Teams app",
+  "core.createProjectQuestion.projectType.copilotHelp.detail": "Chat with GitHub Copilot to get started for your Teams app and agent",
```

Comment:
Chat with GitHub Copilot to get started on your Teams app and agent.

Reply:
Thanks, updated